aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -733,19 +724,17 @@ module aptos_framework::stake {
         operator: &signer,
         pool_address: address,
         new_consensus_pubkey: vector<u8>,
-        proof_of_poseesion: vector<u8>,
     ) acquires StakePool, ValidatorConfig {
-        rotate_consensus_key_internal(operator, pool_address, new_consensus_pubkey, proof_of_poseesion, true);
+        rotate_consensus_key_internal(operator, pool_address, new_consensus_pubkey, true);
     }
 
     /// Rotate the consensus key of the validator, it'll take effect in next epoch.
     public entry fun rotate_consensus_key(
         operator: &signer,
         pool_address: address,
         new_consensus_pubkey: vector<u8>,
-        proof_of_possession: vector<u8>,
     ) acquires StakePool, ValidatorConfig {
-        rotate_consensus_key_internal(operator, pool_address, new_consensus_pubkey, proof_of_possession, false);
+        rotate_consensus_key_internal(operator, pool_address, new_consensus_pubkey, false);
```

Comment:
Again, is `true`, `false` as last parameter required? @so-kkroy22 @so-schen

Reply:
done https://github.com/Entropy-Foundation/aptos-core/pull/9/commits/3fefc82a2c107f84b89b654ccb012cbdd6c7d3a1

Reply:
yes, I think bool flag can be removed